src/Pool.sol

```diff
@@ -773,6 +791,75 @@ abstract contract Pool {
         return subSenders[senderAddr][subSenderId].receiversHash;
     }
 
+    /// @notice Sets a new list of drips receivers of a user
+    /// @param userAddr The user address
+    /// @param currReceivers The list of the user's drips receivers which is currently in use.
+    /// If this function is called for the first time for the user, should be an empty array.
+    /// @param newReceivers The new list of the user's drips receivers.
+    /// Must be sorted by the drips receivers' addresses, deduplicated and without 0 weights.
+    /// Each drips receiver will be getting `weight / TOTAL_DRIPS_WEIGHTS`
+    /// share of the funds collected by the user.
+    function _setDripsReceiversInternal(
+        address userAddr,
+        DripsReceiver[] calldata currReceivers,
+        DripsReceiver[] calldata newReceivers
+    ) internal {
+        _assertCurrDripsReceivers(userAddr, currReceivers);
```

Comment:
We don't care about the currReceiver in my opinion. We can just overwrite it. 

However, should we not check that the `collect` is up to date. I guess that will be in the other PRs :)

Reply:
It'll be important in the next PR :) Here it just enforces the correct usage of the API, which doesn't do any meaningful work yet.